smartsim/_core/control/job.py

```diff
@@ -63,9 +63,12 @@ def __init__(
         self.jid = job_id
         self.entity = entity
         self.status = STATUS_NEW
-        self.raw_status: t.Optional[str] = None  # status before smartsim status mapping is applied
+        self.raw_status: t.Optional[
+            str
+        ] = None  # status before smartsim status mapping is applied
         self.returncode: t.Optional[int] = None
-        self.output: t.Optional[str] = None  # only populated if it's system related (e.g. a command failed immediately)
+        # output is only populated if it's system related (e.g. cmd failed immediately)
+        self.output: t.Optional[str] = None
```

Comment:
minor stylistic: I like the approach you did to split this line far more than the change on the previous line